source/tutorial/ember-data.md

```diff
@@ -0,0 +1,121 @@
+Currently, we're using hard-coded data in the `rentals` route handler file to set the model.  This is good for demonstrating how routing and templates use model information, but very limited in practice.  As our application grows, we will want to be able to create new rentals, make updates to them, and delete them. Ember integrates with a model data management library called Ember Data that will provide this functionality.
+
+Let's generate our first Ember Data model:
+
+```shell
+ember g model rental
+```
+
+which results in the creation of a model file and a test file:
+
+```shell
+installing model
+  create app/models/rental.js
+installing model-test
+  create tests/unit/models/rental-test.js
+```
+
+When we open the model file, we see :
+
+```app/models/rental.js
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+
+});
+```
+
+Let's add the same attributes for our rental that we used in our hard-coded array of JavaScript objects - owner, city, type, image, and bedrooms:
+
+```app/models/rental.js
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  title: DS.attr(),
+  owner: DS.attr(),
+  city: DS.attr(),
+  type: DS.attr(),
+  image: DS.attr(),
+  bedrooms: DS.attr()
+});
+```
+
+Now we have a model in our Ember Data store.
+
+## Using Mirage with Ember Data
+
+Ember Data can be configured to save to any persistent data source. By default, it expects to use a server that you would have to set up yourself. So that we don't have to build a server as part of this tutorial, we are going to configure our app to use [Mirage](http://www.ember-cli-mirage.com) which will help us produce fake data easily for our system.
```

Comment:
> Ember Data can be configured to save to any persistent data source. By default, it expects to use a server that you would have to set up yourself.

Which means in English? ... ;-)  How about:

Ember Data can be configured to save data in a variety of ways, but often is setup to work with a backend API server.